Dockerfile

```diff
@@ -1,8 +1,23 @@
+FROM golang:1.12-alpine as build
```

Comment:
[nit] we should bump this version (if we can, it looks like the code hasn't seem much love lately)

Reply:
Changed to 1.15. More recent versions requre additional rework - and I am not golang dev.